src/main/java/org/spongepowered/api/item/inventory/ContainerTypes.java

```diff
@@ -34,30 +34,44 @@
 import org.spongepowered.api.registry.RegistryScopes;
 
 /**
- * An enumeration of all possible {@link ContainerType}s in vanilla minecraft.
+ * An enumeration of all possible {@link ContainerType}s in vanilla Minecraft.
  */
 @SuppressWarnings("unused")
 @RegistryScopes(scopes = RegistryScope.GAME)
-public final class ContainerTypes {
+to
```

Comment:
what

Reply:
misc local change that made it though lol